Library/Styles/CommentCellStyles.swift

```diff
@@ -27,6 +27,18 @@ public let commentBodyTextViewStyle: TextViewStyle = { textView in
     |> \.dataDetectorTypes .~ .link
 }
 
+// Internationalize
+
+public let collaboratorAuthorBadgeStyle: PaddingLabelStyle = { label in
+  label
+    |> \.text .~ localizedString(key: "Collaborator", defaultValue: "Collaborator")
```

Comment:
Do we not have internationalized `Strings.` for this? If not, please add a `TODO: Internationalize`

Reply:
No, I couldn't fine one.